dataland-frontend/tests/e2e/specs/prepopulation/Prepopulation.ts

```diff
@@ -80,8 +80,8 @@ describe(
             },
             () => {
               const expectedNumberOfCompanies = fixtureData.length;
-              cy.wait(5000);
-              cy.getKeycloakToken(admin_name, admin_pw)
+              cy.wait(5000)
+                .getKeycloakToken(admin_name, admin_pw)
```

Comment:
what is the difference?

Reply:
there shouldnt be any different. Probably a 'wrongly' reverted change